conjur-oss/README.md

```diff
@@ -48,9 +49,9 @@ Conjur Open Source is part of the CyberArk Privileged Access Security Solution w
   * Kubernetes and Helm access to the cluster/namespace is limited to
     security administrators via Role-Based Access Control (RBAC).
 - Kubernetes 1.7+
+- OCP 4.x+
```

Comment:
we are going to need to add a note here that this is community support for now, until our OCP clusters are able to support persistent volumes for automated testing.
```suggestion
- OCP 4.x+ (**Preliminary [Community level](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community) support) only**)
```